cloudformation/media-atom-maker.json

```diff
@@ -345,17 +344,23 @@
                                         ]
                                     ]
                                 },
-                                "aws s3 cp 's3://atom-maker-conf/media-atom-maker.",
+                                "/opt/features/native-packager/install.sh -b atom-maker-dist -a media-atom-maker -t tgz\n",
+
+                                "aws s3 cp 's3://atom-maker-conf/",
                                 {
-                                    "Ref": "Stage"
+                                  "Ref": "Stage"
                                 },
-                                ".application.conf' ",
-                                "/home/media-service/media-atom-maker/conf/application.conf\n",
-                                "cat >> /home/media-service/media-atom-maker/conf/application.conf<<'EOF'\n",
+                                "/media-atom-maker.private.conf' /tmp\n",
+                                "cat >> /tmp/media-atom-maker.private.conf<<'EOF'\n",
```

Comment:
minor question: why cat to the existing file rather than a new file and `include` it?